src/phoenix/server/api/types/GenerativeProvider.py

```diff
@@ -106,6 +106,9 @@ async def dependencies_installed(self) -> bool:
 
     @strawberry.field(description="The credential requirements for the provider")  # type: ignore
     async def credential_requirements(self) -> list[GenerativeProviderCredentialConfig]:
+        # Handle providers that don't require credentials
+        if self.key == GenerativeProviderKey.OLLAMA:
+            return []
         return [self.model_provider_to_credential_requirements_map[self.key]]
```

Comment:
```suggestion
        # Handle providers that don't require credentials
        credential_requirements = self.model_provider_to_credential_requirements_map.get(self.key)
        if credential_requirements is None:
            return []
        return [credential_requirements]
```

Let's avoid an explicit hard-coding of Ollama if it is understood that providers missing from `model_provider_to_credential_requirements_map` do not require credentials. That way we don't have to hard-code such providers in the future.